spark-3.1-spanner-lib/src/test/resources/db/insert_data.sql

```diff
@@ -56,3 +56,21 @@ VALUES
         "id2", [20, 200, 2991, 888885], ["A", "B", "C"], "this one", 93411, DATE("2023-09-23T00:00:00Z"),
         TIMESTAMP("2023-09-22T12:22:05Z"), false, [DATE("2023-09-02T00:00:00Z"), DATE("2023-12-31T00:00:00Z")],
```

Comment:
What's the behavior if customer input a Date("2023-09-29", some_timezone) . I ask because what spanner stores is independent with the https://screenshot.googleplex.com/3pjFRggRNCePsQv . If that's true, I think we can make our test simpler, right ?